Makefile

```diff
@@ -182,3 +182,8 @@ cover: .coverprofile-all ## coverage test and show it in your browser
 .PHONY: show-cover
 show-cover: .coverprofile-all
 	go tool cover -html .coverprofile-all
+
+.PHONY: check-coverage
+check-coverage: ## check 100% test coverage of the selected packages
+	go test -count=1 ./routing/ -coverprofile profile.cov
+	for p in $$(go tool cover -func=profile.cov | grep endpointregistry.go | awk ' { print $$3 } '); do if [ "a$$p" != "a100.0%" ]; then exit 1; fi; done
```

Comment:
It's a bit too specific Makefile target to my taste

Reply:
Okay, will create separate PR for such change to limit the scope of the current PR.